File: algorithms/PalindromeChecker/PalindromeChecker.py
num = input('Enter any number : ')
try:
    val = int(num)
    if num == str(num)[::-1]:
        print('The given number is PALINDROME')
    else:
        print('The given number is NOT a palindrome')
except ValueError:
    print("That's not a valid number, Try Again !")

File: algorithms/PalindromeChecker/PalindromeChecker.c
#include <stdio.h>

void main()
{
    int n,a,sum,t;
    printf("enter any number ");
    scanf("%d",&n);
    sum=0;
    t=n;
    while(n>0 )
    {
         a=n%10;
         n=n/10;
         sum=(sum*10)+a;
    }
         if(t==sum)
             printf("number is palindrome");
         else
             printf("number is not palindrome");

}
